### PR TITLE
Add reminder about ignoring *.backup files via .npmignore

### PR DESCRIPTION
**Pull request by Snaptags**
There should be a reminder about adding an entry to `.npmignore`. Otherwise people might add the backup file to their packed packages and probably won't even notice it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,12 @@ The `clean-package` tool works directly on the 'package.json' file, to avoid bre
 }
 ```
 
+Edit your `.npmignore` file (or add one) and be sure to ignore the backup file. For example:
```
Comment (roydukkey) [inline README.md]:
```suggestion
When the "prepack" script executes, a backup of the original `package.json` will be created. Ensure this file doesn't make it into your release package.

One way to accomplish this is to add the following to your `.npmignore` file:

```

File: README.md [hunk 2]
```diff
@@ -28,6 +28,12 @@ The `clean-package` tool works directly on the 'package.json' file, to avoid bre
...
+```text
+# Build Artifacts
```
Comment (roydukkey) [inline README.md]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -28,6 +28,12 @@ The `clean-package` tool works directly on the 'package.json' file, to avoid bre
 }
 ```
 
+Edit your `.npmignore` file (or add one) and be sure to ignore the backup file. For example:
+```text
```
Comment (roydukkey) [inline README.md]:
```suggestion
```ignore
```

**Review by roydukkey — CHANGES_REQUESTED** [review]
How do these changes look?

**Comment by Snaptags** [comment]
Thanks for reviewing! I like your suggested changes and applied them.

**Comment by roydukkey** [comment]
Thanks!

**Pull request merged**